gubernator/config.yaml

```diff
@@ -34,6 +34,7 @@ jobs:
   - pull-kubernetes-e2e-kind
   - pull-kubernetes-e2e-kind-ipv6
   - pull-kubernetes-integration
+  - pull-kubernetes-make-test
```

Comment:
Silly nit I didn't catch before the PR introducing this job merged: WDYT about `pull-kubernetes-test` or `pull-kubernetes-unit` instead? Avoid baking the name of how they're invoked into the job

Reply:
+1 for pull-kubernetes-unit

Reply:
Sure

Reply:
done